apps/site/src/pages/settings/billing-settings-panel/payment-methods.tsx

```diff
@@ -217,7 +217,28 @@ export const PaymentMethodsPanelPage: FunctionComponent = () => {
               >
                 <PaymentMethod
                   paymentMethod={paymentMethod}
-                  isDefault={isDefault}
+                  sx={{ marginY: 2 }}
+                  titleEndAdornment={
+                    <Fade in={isDefault}>
+                      <Box
+                        component="span"
+                        sx={({ palette, spacing }) => ({
+                          padding: spacing(0.5, 0.75),
+                          color: palette.purple[700],
+                          background: palette.purple[20],
+                          borderRadius: 1,
+                          textTransform: "uppercase",
+                          fontSize: 11,
+                        })}
+                      >
+                        <FontAwesomeIcon
+                          icon={faCheck}
+                          sx={{ marginRight: 0.75, fontSize: 12 }}
+                        />
+                        <strong>Default</strong>
+                      </Box>
+                    </Fade>
+                  }
                 />
                 <PaymentMethodMenu
```

Comment:
Non-blocking: Could this be placed in `endAdornment` like we are doing for the edit button in the modal?

Reply:
If you're referring to the "Default" chip here, then no because the chip needs to be positioned next to the name of the card. That's why I added the separate `titleEndAdornment` prop.

Reply:
I was referring to this button:
![image](https://user-images.githubusercontent.com/37453800/217671252-89e2c1ec-9314-43eb-9911-995e47b756d6.png)